plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -8261,6 +8270,52 @@ public void testSelectFromPrestoViewReferencingHiveTableWithTimestamps()
         assertThat(query(nanosSessions, "SELECT ts FROM hive_timestamp_nanos.tpch." + prestoViewNameNanos)).matches("VALUES TIMESTAMP '1990-01-02 12:13:14.123000000'");
     }
 
+    @Test
+    public void testTimestampWithTimeZone()
+    {
+        assertUpdate("CREATE TABLE test_timestamptz_base (t timestamp) WITH (format = 'PARQUET')");
+        assertUpdate("INSERT INTO test_timestamptz_base (t) VALUES" +
+                     "(timestamp '2022-07-26 12:13')", 1);
+
+        // Writing TIMESTAMP WITH LOCAL TIME ZONE is not supported, so we first create Parquet object by writing unzoned
+        // timestamp (which is converted to UTC using default timezone) and then creating another table that reads from the same file.
+        String tableLocation = (String) computeActual("SELECT DISTINCT regexp_replace(\"$path\", '/[^/]*$', '') FROM test_timestamptz_base").getOnlyValue();
```

Comment:
It'd be more robust to create the table as an external table and provide a temporary directory, instead. Then you can create the other table pointing at the same directory.

Reply:
I replaced this call with `getTableLocation`, which is what other tests in this file use. This is a different solution than what you suggested, but seems more consistent (let me know if you disagree).

Reply:
That works, too. Thanks!